### PR TITLE
Update the Command Tree for Vulkan.

### PR DESCRIPTION
**Pull request by AWoloszyn**
This slightly re-works the command tree to be significantly
more explicit about what is run on CPU-side and what is run on
the GPU.

### CODE REVIEW

File: gapic/src/main/com/google/gapid/widgets/ImagePanel.java [hunk 1]
```diff
@@ -1279,7 +1279,7 @@ public void accept(double value) {
 
         for (int bin = 0; bin < numBins; bin++) {
           float value = data.histogram.get(channel, bin);
-          value = (float)Math.pow(value, 0.3); // Exaggerate small values.
```
Comment (pmuetschard) [inline gapic/src/main/com/google/gapid/widgets/ImagePanel.java]: unrelated change

Comment (AWoloszyn) [reply gapic/src/main/com/google/gapid/widgets/ImagePanel.java]: Thanks, I thought I had removed that change, but obviously not.

Comment (AWoloszyn) [reply gapic/src/main/com/google/gapid/widgets/ImagePanel.java]: Fixed
